<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShoppingManager</class>
 <widget class="QWidget" name="ShoppingManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="mainShoppingLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome to OssMall</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="userIdLoginLineEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="shoppingLoginPushButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>로그인</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addNewClientPushButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>회원가입</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeClientPushButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>회원탈퇴</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="productInfoLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>26</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>제품 정보</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="productInfoTreeWidget">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="autoScrollMargin">
            <number>16</number>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideRight</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <column>
            <property name="text">
             <string>제품 번호</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 이름</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 가격</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 수량</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 종류</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="orderListLabel">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>26</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>주문 내역</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="chatClientPushButton">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>채팅하기</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chatServerPushButton">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>서버오픈</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTreeWidget" name="orderListTreeWidget">
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideRight</enum>
           </property>
           <column>
            <property name="text">
             <string>주문 번호</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 이름</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 가격</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>주문 수량</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>제품 종류</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>배송 주소</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="takeOrderPushButton">
           <property name="text">
            <string>주문하기(Take Order)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="updateOrderPushButton">
             <property name="text">
              <string>주문변경(Change)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelOrderPushButton">
             <property name="text">
              <string>주문취소(Cancel)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="managementPushButton">
           <property name="text">
            <string>제품/회원 정보 관리(Only Staff)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exitShoppingPushButton">
           <property name="text">
            <string>쇼핑 끝내기(Exit)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userIdLoginLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>shoppingLoginPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
